### ISSUE TITLE
Add overlay from Swedish National Road Database (NVDB)

### ISSUE THREAD

**Issue by magol** [osmlab/editor-layer-index#352]
## URL of the imagery

1. https://mapproxy.openstreetmap.se/tiles/1.0.0/nvdb_speeds/EPSG3857/{zoom}/{x}/{y}.png
2. https://mapproxy.openstreetmap.se/tiles/1.0.0/nvdb_speeds_alt/EPSG3857/{zoom}/{x}/{y}.png
3. https://mapproxy.openstreetmap.se/tiles/1.0.0/nvdb_names/EPSG3857/{zoom}/{x}/{y}.png

## Two letter country code
sv

## Desired layer name

1. NVDB - Speed and cycle paths
2. NVDB - Speed and cycle paths (alternative colors)
3. NVDB - Highway names

## License
CC0

## Is the license compatible with OSM?
Yes

## Attribution (text, URL, required?)
Text: Lantmäteriet
Url: [lantmateriet.se](lantmateriet.se)
Required: Yes

## Icon
<img src="http://www.trafikverket.se/contentassets/b7724f5b14b545b2832a4c749c6f9404/profilbild.png" width="64" title="Lantmäteriet"/>

## Boundary polygon
```
<?xml version="1.0" encoding="UTF-8" ?>
<!-- Generated with JOSM Imagery XML Plugin version 12443 -->
<imagery xmlns="http://josm.openstreetmap.de/maps-1.0">
    <entry>
        <name>Sverige</name>
        <type>wms</type>
        <url></url>
        <bounds min-lat='55.1331192' min-lon='10.5920778' max-lat='69.0599699' max-lon='24.1776819'>
            <shape>
                <point lat='57.3348282' lon='17.6814016'/><point lat='57.3216162' lon='17.6794646'/><point lat='57.3083647' lon='17.6791301'/>
                <point lat='57.2951292' lon='17.6803971'/><point lat='57.2819667' lon='17.6832591'/><point lat='57.2689337' lon='17.6877021'/>
                <point lat='57.2560862' lon='17.6937051'/><point lat='57.2434782' lon='17.7012411'/><point lat='57.2311637' lon='17.7102766'/>
                <point lat='57.2270662' lon='17.7136716'/><point lat='57.2020987' lon='17.7324381'/><point lat='57.0506117' lon='17.7823836'/>
                <point lat='56.9464617' lon='17.7577226'/><point lat='56.9433747' lon='17.7570381'/><point lat='56.9301642' lon='17.7550951'/>
                <point lat='56.9169122' lon='17.7547376'/><point lat='56.9036757' lon='17.7559646'/><point lat='56.8905117' lon='17.7587706'/>
                <point lat='56.8774762' lon='17.7631406'/><point lat='56.8646247' lon='17.7690551'/><point lat='56.8520122' lon='17.7764871'/>
                <point lat='56.8396922' lon='17.7854031'/><point lat='56.8277177' lon='17.7957636'/><point lat='56.8161392' lon='17.8075246'/>
                <point lat='56.8050057' lon='17.8206326'/><point lat='56.7943652' lon='17.8350321'/><point lat='56.7842622' lon='17.8506616'/>
                <point lat='56.7747392' lon='17.8674521'/><point lat='56.7658372' lon='17.8853326'/><point lat='56.7575937' lon='17.9042271'/>
                <point lat='56.7500432' lon='17.9240546'/><point lat='56.7432182' lon='17.9447311'/><point lat='56.7371477' lon='17.9661686'/>
                <point lat='56.7318562' lon='17.9882756'/><point lat='56.7286417' lon='18.0040291'/><point lat='56.7153777' lon='18.0737146'/>
                <point lat='56.7141022' lon='18.0806216'/><point lat='56.7140987' lon='18.0806401'/><point lat='56.7104172' lon='18.1037896'/>
                <point lat='56.7075717' lon='18.1273241'/><point lat='56.7055742' lon='18.1511431'/><point lat='56.7044332' lon='18.1751471'/>
                <point lat='56.7041537' lon='18.1992341'/><point lat='56.7047367' lon='18.2233036'/><point lat='56.7061797' lon='18.2472546'/>
                <point lat='56.7084762' lon='18.2709841'/><point lat='56.7116172' lon='18.2943941'/><point lat='56.7123507' lon='18.2990571'/>
                <point lat='56.7272732' lon='18.3919451'/><point lat='56.7304982' lon='18.4102876'/><point lat='56.7352677' lon='18.4327831'/>
                <point lat='56.7408317' lon='18.4546691'/><point lat='56.7471667' lon='18.4758531'/><point lat='56.7542462' lon='18.4962456'/>
                <point lat='56.7620402' lon='18.5157591'/><point lat='56.7705152' lon='18.5343121'/><point lat='56.7796362' lon='18.5518236'/>
                <point lat='56.7801232' lon='18.5526966'/><point lat='56.8183102' lon='18.6210391'/><point lat='56.8275427' lon='18.6365956'/>
                <point lat='56.8378277' lon='18.6518406'/><point lat='56.8433157' lon='18.6591876'/><point lat='56.8777092' lon='18.7036866'/>
                <point lat='57.1032907' lon='19.0018981'/><point lat='57.3077572' lon='19.2871866'/><point lat='57.3170557' lon='19.2995081'/>
                <point lat='57.3282752' lon='19.3125471'/><point lat='57.3399307' lon='19.3242096'/><point lat='57.3519727' lon='19.3344441'/>
                <point lat='57.3643502' lon='19.3432066'/><point lat='57.3770102' lon='19.3504571'/><point lat='57.3898997' lon='19.3561646'/>
                <point lat='57.4029622' lon='19.3603016'/><point lat='57.4077332' lon='19.3614106'/><point lat='57.4243897' lon='19.3649121'/>
                <point lat='57.4327997' lon='19.3663531'/><point lat='57.4460422' lon='19.3673006'/><point lat='57.4592897' lon='19.3666401'/>
                <point lat='57.4724857' lon='19.3643726'/><point lat='57.4855732' lon='19.3605056'/><point lat='57.4984967' lon='19.3550541'/>
                <point lat='57.5111997' lon='19.3480401'/><point lat='57.5236282' lon='19.3394921'/><point lat='57.5357287' lon='19.3294451'/>
                <point lat='57.5474487' lon='19.3179411'/><point lat='57.5587382' lon='19.3050281'/><point lat='57.5675582' lon='19.2935386'/>
                <point lat='57.6463957' lon='19.3999911'/><point lat='57.6469622' lon='19.4007551'/><point lat='57.8396637' lon='19.6615451'/>
                <point lat='57.8444797' lon='19.6679091'/><point lat='57.8556597' lon='19.6812541'/><point lat='57.8672797' lon='19.6932071'/>
                <point lat='57.8792907' lon='19.7037156'/><point lat='57.8916417' lon='19.7127336'/><point lat='57.9042797' lon='19.7202216'/>
                <point lat='57.9171512' lon='19.7261451'/><point lat='57.9302012' lon='19.7304781'/><point lat='57.9433737' lon='19.7332001'/>
                <point lat='57.9566127' lon='19.7342971'/><point lat='57.9698617' lon='19.7337631'/><point lat='57.9830637' lon='19.7315991'/>
                <point lat='57.9961617' lon='19.7278111'/><point lat='58.0091002' lon='19.7224151'/><point lat='58.0218232' lon='19.7154326'/>
                <point lat='58.0342762' lon='19.7068906'/><point lat='58.0464052' lon='19.6968251'/><point lat='58.0581582' lon='19.6852776'/>
                <point lat='58.0694847' lon='19.6722971'/><point lat='58.0803352' lon='19.6579381'/><point lat='58.0885502' lon='19.6456431'/>
                <point lat='58.1169862' lon='19.6007111'/><point lat='58.1190977' lon='19.5973246'/><point lat='58.1288522' lon='19.5803706'/>
                <point lat='58.1379967' lon='19.5622386'/><point lat='58.1464912' lon='19.5430051'/><point lat='58.1542997' lon='19.5227531'/>
                <point lat='58.1613872' lon='19.5015706'/><point lat='58.1654892' lon='19.4873156'/><point lat='58.1692322' lon='19.4989046'/>
                <point lat='58.1768647' lon='19.5194071'/><point lat='58.1851862' lon='19.5389311'/><point lat='58.1941617' lon='19.5573916'/>
                <point lat='58.2037527' lon='19.5747101'/><point lat='58.2139197' lon='19.5908131'/><point lat='58.2246187' lon='19.6056306'/>
                <point lat='58.2246632' lon='19.6056881'/><point lat='58.2296582' lon='19.6121411'/><point lat='58.2382107' lon='19.6255656'/>
                <point lat='58.2489052' lon='19.6404036'/><point lat='58.2600877' lon='19.6538916'/><point lat='58.2717097' lon='19.6659716'/>
                <point lat='58.2837222' lon='19.6765906'/><point lat='58.2872402' lon='19.6793656'/><point lat='58.2892082' lon='19.6808776'/>
                <point lat='58.2980422' lon='19.6872146'/><point lat='58.3106812' lon='19.6947791'/><point lat='58.3235532' lon='19.7007616'/>
                <point lat='58.3366032' lon='19.7051346'/><point lat='58.3497752' lon='19.7078781'/><point lat='58.3630137' lon='19.7089781'/>
                <point lat='58.3762612' lon='19.7084291'/><point lat='58.3894622' lon='19.7062301'/><point lat='58.4025587' lon='19.7023896'/>
                <point lat='58.4154952' lon='19.6969221'/><point lat='58.4282157' lon='19.6898496'/><point lat='58.4406657' lon='19.6812006'/>
                <point lat='58.4527922' lon='19.6710106'/><point lat='58.4645417' lon='19.6593221'/><point lat='58.4758642' lon='19.6461836'/>
                <point lat='58.4867107' lon='19.6316511'/><point lat='58.4970347' lon='19.6157866'/><point lat='58.5067907' lon='19.5986561'/>
                <point lat='58.5159367' lon='19.5803346'/><point lat='58.5228492' lon='19.5647246'/><point lat='58.5236307' lon='19.5628621'/>
                <point lat='58.5238887' lon='19.5623781'/><point lat='58.5330297' lon='19.5440381'/><point lat='58.5415207' lon='19.5245846'/>
                <point lat='58.5493252' lon='19.5041021'/><point lat='58.5564092' lon='19.4826781'/><point lat='58.5627417' lon='19.4604046'/>
                <point lat='58.5682952' lon='19.4373801'/><point lat='58.5716402' lon='19.4212111'/><point lat='58.5728247' lon='19.4167616'/>
                <point lat='58.5783702' lon='19.3937236'/><point lat='58.5831127' lon='19.3700331'/><point lat='58.5870317' lon='19.3457941'/>
                <point lat='58.5901097' lon='19.3211111'/><point lat='58.5923332' lon='19.2960926'/><point lat='58.5927842' lon='19.2893216'/>
                <point lat='58.5949417' lon='19.2712511'/><point lat='58.5971557' lon='19.2462261'/><point lat='58.5985057' lon='19.2209746'/>
                <point lat='58.5985657' lon='19.2192616'/><point lat='58.5991417' lon='19.2064056'/><point lat='58.5996187' lon='19.1810381'/>
                <point lat='58.5992242' lon='19.1556646'/><point lat='58.5979587' lon='19.1303966'/><point lat='58.5958287' lon='19.1053441'/>
                <point lat='58.5949342' lon='19.0971141'/><point lat='58.5944357' lon='19.0927656'/><point lat='58.5923437' lon='19.0762676'/>
                <point lat='58.5885152' lon='19.0519726'/><point lat='58.5838607' lon='19.0282151'/><point lat='58.5784002' lon='19.0050981'/>
                <point lat='58.5721582' lon='18.9827226'/><point lat='58.5651612' lon='18.9611846'/><point lat='58.5574402' lon='18.9405786'/>
                <point lat='58.5490282' lon='18.9209916'/><point lat='58.5399622' lon='18.9025081'/><point lat='58.5302812' lon='18.8852086'/>
                <point lat='58.5234867' lon='18.8743406'/><point lat='58.5217602' lon='18.8717001'/><point lat='58.5182997' lon='18.8665261'/>
                <point lat='58.5075167' lon='18.8518106'/><point lat='58.4962517' lon='18.8384831'/><point lat='58.4845532' lon='18.8266001'/>
                <point lat='58.4724722' lon='18.8162106'/><point lat='58.4600597' lon='18.8073586'/><point lat='58.4473707' lon='18.8000801'/>
                <point lat='58.4344582' lon='18.7944051'/><point lat='58.4213792' lon='18.7903561'/><point lat='58.4093132' lon='18.7880886'/>
                <point lat='58.4063407' lon='18.7877016'/><point lat='58.4052167' lon='18.7875611'/><point lat='58.3919722' lon='18.7868036'/>
                <point lat='58.3787297' lon='18.7876976'/><point lat='58.3655462' lon='18.7902371'/><point lat='58.3524782' lon='18.7944101'/>
                <point lat='58.3395812' lon='18.8001951'/><point lat='58.3269107' lon='18.8075681'/><point lat='58.3145202' lon='18.8164951'/>
                <point lat='58.3024622' lon='18.8269361'/><point lat='58.2907887' lon='18.8388451'/><point lat='58.2903637' lon='18.8393146'/>
                <point lat='58.2886377' lon='18.8408166'/><point lat='58.2879017' lon='18.8413981'/><point lat='58.2859507' lon='18.8423751'/>
                <point lat='58.2848027' lon='18.8429566'/><point lat='58.2721297' lon='18.8503026'/><point lat='58.2597357' lon='18.8592006'/>
                <point lat='58.2476742' lon='18.8696111'/><point lat='58.2359967' lon='18.8814881'/><point lat='58.2247522' lon='18.8947801'/>
                <point lat='58.2139887' lon='18.9094286'/><point lat='58.2037517' lon='18.9253706'/><point lat='58.1940852' lon='18.9425381'/>
                <point lat='58.1850292' lon='18.9608571'/><point lat='58.1785997' lon='18.9756881'/><point lat='58.1265707' lon='18.6230441'/>
                <point lat='58.1231317' lon='18.6023356'/><point lat='58.1183942' lon='18.5789466'/><point lat='58.1128532' lon='18.5561996'/>
                <point lat='58.1065332' lon='18.5341946'/><point lat='58.0994617' lon='18.5130246'/><point lat='58.0916687' lon='18.4927836'/>
                <point lat='58.0843065' lon='18.4760909'/><point lat='58.0812171' lon='18.4625461'/><point lat='58.0831627' lon='18.4422678'/>
                <point lat='58.0751134' lon='18.4229494'/><point lat='58.0638443' lon='18.4004112'/><point lat='58.041306' lon='18.3617742'/>
                <point lat='57.9495433' lon='18.3005991'/><point lat='57.9044669' lon='18.2523029'/><point lat='57.8497313' lon='18.176639'/>
                <point lat='57.7870873' lon='18.0549496'/><point lat='57.7363318' lon='17.9582275'/><point lat='57.7096724' lon='17.8723731'/>
                <point lat='57.6342782' lon='17.7794741'/><point lat='57.6224817' lon='17.7682151'/><point lat='57.6114697' lon='17.7592791'/>
                <point lat='57.6107832' lon='17.7587671'/><point lat='57.6096287' lon='17.7579131'/><point lat='57.5971432' lon='17.7496331'/>
                <point lat='57.5843942' lon='17.7428966'/><point lat='57.5714347' lon='17.7377316'/><point lat='57.5583217' lon='17.7341586'/>
                <point lat='57.5554712' lon='17.7335966'/><point lat='57.4574037' lon='17.7156181'/><point lat='57.3425932' lon='17.6832971'/>
                <point lat='57.3348282' lon='17.6814016'/>
            </shape>
        </bounds>
    </entry>
</imagery>
```

**Comment by tyrasd** [comment]
* What's the coverage area of this layer?
* The `{z}` parameter should instead read `{zoom}` to be fully compatible with the schema in this repository.

**Comment by magol** [comment]
@tyrasd It covers the whole Sweden.

**Comment by tyrasd** [comment]
Yes, we can see that from the ticket title. :smirk: What I meant is that it would be great to have the exact outline of the imagery coverage either as a GeoJSON polygon (or a similar spatial data format), or at least the bounding box coordinates of it. I think one can use JOSM to determine these data (I think there's a page on the osm wiki explaining it). Alternatively, there are some other tools that might come in handy, for example @nrenner's bbox tool: http://norbertrenner.de/osm/bbox.html or http://geojson.io/.

**Comment by magol** [comment]
I did use[ Imagery-XML-Bounds](https://wiki.openstreetmap.org/wiki/JOSM/Plugins/Imagery-XML-Bounds) to get the XML Bounds for the [Swedish multipolygon](http://www.openstreetmap.org/relation/52822). I hope it is correct

**Comment by grischard** [comment]
Ok, let's  get this one in :). What's the url for the CC-0 license?

**Comment by grischard** [comment]
Oh, and it looks like the icon is broken

**Comment by grischard** [comment]
Fixed in 4f2d324
